src/go/rpk/pkg/cli/redpanda/admin/config/config.go

```diff
@@ -185,13 +185,15 @@ var possibleLoggers = []string{
 	"compression",
 	"controller_rate_limiter_log",
 	"cpu_profiler",
+	"datalake",
```

Comment:
Huh, FWIW I'm pretty sure we have an endpoint for this list in the admin endpoint

Reply:
Oh heh didn't see the comment